language/German/strings.po

```diff
@@ -2227,12 +2271,12 @@ msgstr "Vorschau Illustrationen"
 #: src/frontend/mame/ui/dirmenu.cpp:69
 msgctxt "path-option"
 msgid "Select"
-msgstr ""
+msgstr "Auswahl"
```

Comment:
"Auswählen" (it's a verb, the noun would be "selection")

Reply:
From the context I think my proposal is still better thought, so I won't change that.

Reply:
It’s a folder for screenshots showing character selection screens.  It isn’t an imperative in the English.

Reply:
you're right I didn't fully check the context (it was rather late ;-)

Reply:
Ok, I'll change it to _Charakterauswahl_ then